src/abstractions/node/gitCredentialFill.ts

```diff
@@ -1,4 +1,27 @@
 import { execSync } from "child_process";
+import * as path from "path";
+import * as vscode from "vscode";
+
+function getAskPassEnv(useAskPassEmpty = true) {
+  const gitExt = vscode.extensions.getExtension("vscode.git");
+  const gitExtPath = gitExt?.extensionPath;
+
+  if (!gitExtPath) {
+    return {};
+  }
+
+  if (useAskPassEmpty) {
+    return {
+      GIT_ASKPASS: path.join(gitExtPath, "dist", "askpass-empty.sh")
+    };
+  }
+
+  return {
+    GIT_ASKPASS: path.join(gitExtPath, "dist", "askpass.sh"),
+    VSCODE_GIT_ASKPASS_NODE: process.execPath,
+    VSCODE_GIT_ASKPASS_MAIN: path.join(gitExtPath, "dist", "askpass.sh")
+  };
+}
```

Comment:
Since the only part that seems to work in this so far is the `askpass-empty.sh`, an `askpass-empty,sh` script could just be added to this extension, instead of relying on the git extension, because all it does is `echo ''`. The part where I attempt to set `VSCODE_GIT_ASKPASS_NODE` and `VSCODE_GIT_ASKPASS_MAIN` could be removed if it stays readonly, because that is only there from my attempt at getting the login prompt to work.